docs/src/development/email.md

```diff
@@ -92,61 +44,113 @@ file=none
 highlight=false
 ---
 
-To allow emails, run the following command:
+To turn on outgoing email, run the following command:
 
 ```bash
 platform environment:info -e <ENVIRONMENT_NAME> enable_smtp true
 ```
 
-To disallow emails, replace `true` with `false`.
+To turn off outgoing email, replace `true` with `false`.
 
 {{< /codetabs >}}
 
 Changing the setting rebuilds the environment.
 
-## SMTP host address
+## 2. (Optional) Add email domain validation
```

Comment:
:hammer_and_wrench: :arrow_forward:

Maybe we could:
- rename that section to `Improve deliverability`
- start off with the SPF part and then get to the Elite-only DKIM section?

That would make sure that non-elite customers don't skip the SPF part?

Reply:
That was how I had it initially, but I thought it made sense to prioritize the premium offer. Also so it didn't seem like people who do DKIM should also do the other option. But I can try with different headings now.

Reply:
Huh. That wasn't my impression, and I definitely didn't get that from your previous suggestion. I will switch it to better match that idea.

Reply:
Sorry, I should've been more precise in my explanation. The flow is much clearer now 😍